builder/docs_builder.livecodescript

```diff
@@ -1941,6 +1941,12 @@ private command addToList pElement, @xArray
    put pElement into xArray[tCount + 1]
 end addToList
 
+private command addFolderToListIfExists pFolder, @xArray
+   if there is a folder pFolder then
+      addToList pFolder, xArray
+   end if
+end addFolderToListIfExists
```

Comment:
It might be worth making this emit an "info"-level message if it skips a folder due to it not existing. It might help diagnose weirdness happening due to a folder going away.